Shuttlerock/actions
# [STUDIO-581] When a new label is added, remove conflicting labels

## When a new label is added, remove conflicting labels

[Jira Tech task](https://shuttlerock.atlassian.net/browse/STUDIO-581)

## How to test the PR

`$ act --job pull_request_labeled_action --eventpath src/actions/pull-request-labeled-action/__tests__/fixtures/pull-request-labeled.json`

Set the label on this PR to `in-progress` and run the action. Make sure that the labels finish as `please review` and `under-discussion`.

## Deployment Notes

None.
